six.py

```diff
@@ -300,6 +300,7 @@ class _MovedItems(_LazyModule):
     MovedModule("urllib_robotparser", "robotparser", "urllib.robotparser"),
     MovedModule("xmlrpc_client", "xmlrpclib", "xmlrpc.client"),
     MovedModule("xmlrpc_server", "SimpleXMLRPCServer", "xmlrpc.server"),
+    MovedModule("collections_abc", "collections", "collections.abc", sys.version_info[0:2] >= (3, 3)),
```

Comment:
Why not just `sys.version_info >= (3, 3)` instead of `sys.version_info[0:2] >= (3, 3)`?

Reply:
Consistency with the PYX variables at file's beginning